wilds/datasets/poverty_dataset.py

```diff
@@ -236,21 +235,12 @@ def __init__(self, version=None, root_dir='data', download=False,
         super().__init__(root_dir, download, split_scheme)
 
     def get_input(self, idx):
-       """
-       Returns x for a given idx.
-       """
-       img = self.imgs[idx].copy()
-       if self.no_nl:
-           img[-1] = 0
-       img = torch.from_numpy(img).float()
-
-       self.cache_counter += 1
-       if self.cache_counter > 1000:
-           self.imgs = np.load(self.root / 'landsat_poverty_imgs.npy', mmap_mode='r')
-           self.imgs = self.imgs.transpose((0, 3, 1, 2))
-           self.cache_counter = 0
-
-       return img
+        """
+        Returns x for a given idx.
+        """
+        img = np.load(self.root / 'images' / f'landsat_poverty_img_{idx}.npz')['x']
```

Comment:
we're not supporting `no_nl` anymore? the option is still there, i feel that it's easy to keep for anyone that would want to try it / are we keeping this baseline in the paper?